script/update-docs.rb

```diff
@@ -182,7 +183,15 @@ def index_l10n_doc(filter_tags, doc_list, get_content)
         anchor += "-1" while ids.include?(anchor)
         ids.add(anchor)
 
-        "<dt class=\"hdlist1\" id=\"#{anchor}\"> <a class=\"anchor\" href=\"##{anchor}\"></a>#{$1} </dt>"
+        clean_text = Nokogiri::HTML.parse($1).text.tr("^A-Za-z0-9-", "")
+        if clean_text != text
+          clean_anchor = "#{txt_path}-#{clean_text}"
+          clean_anchor += "-1" while ids.include?(clean_anchor)
+          ids.add(clean_anchor)
+          clean_anchor = "<a class=\"anchor\" href=\"##{clean_anchor}\"></a> "
```

Comment:
@dscho LOL, we did it wrong. We're adding another link with a `href`, while we should have added a span (or whatever) with a `id` :D. Can we thank Co-pilot for this nice mixup?

Reply:
:rofl: I was about to make the same remark! A patch is in making.